docs/detections/alert-suppression.asciidoc

```diff
@@ -4,7 +4,9 @@
 .Requirements and notices
 [sidebar]
 --
-Alert suppression requires a https://www.elastic.co/pricing[Platinum or higher subscription].
+* Alert suppression requires a https://www.elastic.co/pricing[Platinum or higher subscription].
```

Comment:
Platinum only for ML I believe. 

@approksiu can you confirm the licensing requirements for suppression?

Reply:
Alert suppression is a platinum feature. So not only for ML.